✅ Test: 지도제외 실종 글타래 조회 테스트

gd06070 opened the pull request:
## 🛠️ 구현한 기능
- 미니맵 제외 데이터 불러옴 (량님한테 미니맵 던짐)

## 📝 구현한 내용
- 구현 내용 1
- 이미지 첨부 (선택)

## ✅ 체크리스트
- [ ] 이슈 내용 모두 적용
- [ ] 작업 기간 내 개발

## 💬 리뷰 요구 사항
- 기능 관련 나누고 싶은 내용 or 리뷰 요구사항 작성

## 🔗 연관된 이슈
- close #76

gd06070 committed:
✅ Test: 지도제외 실종 글타래 조회 테스트 파일  #76

SoRaang reviewed (approved):
pages/Test.jsx 테스트 페이지 추가 확인

ryan-g00 committed:
병합
Merge branch 'develop' of https://github.com/prgrms-web-devcourse-final-project/WEB1_2_Mallangs_FE into feat/#76
